compose.yaml

```
@@ -0,0 +1,27 @@
+services:
+  app:
+    ports:
+      - "8086:80"
+    depends_on:
+      - db
```

Le compose.yaml en l'état ne fonctionnera pas. Modifier avec ceci.

```suggestion
    build: .
    ports:
      - 8086:80
    depends_on:
      - db
    volumes:
      - ./:/var/www/html
```